Mathlib/GroupTheory/GroupAction/Group.lean

```diff
@@ -189,12 +189,6 @@ variable {c : α} {x y : β} [Invertible c]
 @[simp]
 theorem invOf_smul_smul : ⅟c • c • x = x := inv_smul_smul (unitOfInvertible c) _
 
-@[simp]
-theorem smul_invOf_smul : c • (⅟ c • x) = x := smul_inv_smul (unitOfInvertible c) _
```

Comment:
We should keep this one; my claim was just that we should only have the ones which mention `⅟` in the statement, which this does.

Reply:
Okay, I've re-instated that lemma.